client/components/mma/upgrade/UpgradeSupport.stories.tsx

```diff
@@ -15,6 +15,13 @@ export default {
 	parameters: {
 		layout: 'fullscreen',
 		reactRouter: {
+			state: {
```

Comment:
can we remove the properties that aren't actually part of the `UpgradeRouterState` interface?